src/safemode_ui.cpp

```diff
@@ -599,7 +599,7 @@ rule_state safemode::check_monster( const std::string &creature_name_in,
     if( iter != safemode_rules.end() ) {
         const auto &tmp = ( iter->second )[( int )attitude_in];
         if( tmp.state == RULE_BLACKLISTED ) {
-            const int check_dist = ( tmp.proximity == 0 ) ? 50 : tmp.proximity;
+            const int check_dist = ( tmp.proximity == 0 ) ? 60 : tmp.proximity;
```

Comment:
I'm pretty sure it's supposed to be a calculable value (or, at the very least, a named constant). If it ends up with a magic number, it'll break again some (probably not too distant) day.